packages/proto-loader/src/index.ts

```diff
@@ -395,6 +395,15 @@ export function loadSync(
   return createPackageDefinition(root, options!);
 }
 
+export function fromJSON(
+  json: Protobuf.INamespace
+): PackageDefinition {
+  const newRoot: Protobuf.Root = new Protobuf.Root();
```

Comment:
At this point, I don't see any reason to create a new `Root` here. `fromJSON` creates one by default anyway.